src/tests/rotation_points.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
what about a testcase for a point at origin? i.e. position 0,0,0

Reply:
I think it's absolutely zero. in the rotation equation (v' = Ry*Rz*v), when v is zero(v=o) , the v' will be zero and it won't change. but for sure, I add a point for that.

Reply:
you're right. But again, we don't know :D maybe the code makes a stupid mistake. that's the idea of writing testcases.